File: WarehouseManagementWeb.Application/Dto/Output/MeasureUnit/MeasureUnitListByStatusOutput.cs
namespace WarehouseManagementWeb.Application.Dto.Output.MeasureUnit
{
    /// <summary>
    /// Класс выходной модели единиц измерений разделенных по статусам.
    /// </summary>
    public class MeasureUnitListByStatusOutput
    {
        /// <summary>
        /// Список активных единиц измерения.
        /// </summary>
        public IEnumerable<MeasureUnitOutput>? ActiveMeasureUnits { get; set; }

        /// <summary>
        /// Список единиц измерения находящихся в архиве.
        /// </summary>
        public IEnumerable<MeasureUnitOutput>? ArchivedMeasureUnits { get; set; }
    }
}

File: warehouse-management-web-angular/src/app/directory/measure-unit/models/output/measure-unit-list-by-status-output.ts
import { MeasureUnitOutput } from './measure-unit-output';

/**
 * Класс выходной модели единиц измерений разделенных по статусам.
 */
export class MeasureUnitListByStatusOutput {
  /**
   * Список активных единиц измерения.
   */
  activeMeasureUnits: MeasureUnitOutput[] = [];

  /**
   * Список единиц измерения находящихся в архиве.
   */
  archivedMeasureUnits: MeasureUnitOutput[] = [];
}
